nuxeo-runtime/nuxeo-runtime-stream/pom.xml

```diff
@@ -28,6 +28,11 @@
       <artifactId>nuxeo-runtime-test</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.avro</groupId>
+      <artifactId>avro</artifactId>
+      <version>1.8.2</version>
```

Comment:
Is this needed? Version comes from the parent pom.xml.

Reply:
the parent pom property must used indeed, I'll use it (now I know) when Ben merges its PR

Reply:
👍